Inc/Data.h

```diff
@@ -29,6 +29,12 @@ typedef struct
     int32_t     pressure_;
 } CombustionChamberPressureData;
 
+typedef struct
+{
+    osMutexId   mutex_;
+    uint32_t     voltage_;
```

Comment:
What units are we storing `voltage_` as? If undecided, I would suggest mV. This way, we can use an `int16_t` and store a value anywhere from -32.768V to 32.767V, which I think is a reasonable range. I don't think we need a full `uint32_t` for storing voltage lol.